### PR TITLE
Dev setup simplification (saner defaults for dev env)

### PR DESCRIPTION
**Pull request by webplate**
To simplify the installation of a dev setup I propose these light additions for a saner experience: a few cities to test the site with, an admin user to edit the CMS pages and the correct configuration for the mailer application. I would like to make clear that this does not populate the DB unless `rake db:seed:users` and `rake db:seed:cities` are called and that the production config of the mailer remains untouched, it's only meant for developers !

This is related to #164 and https://github.com/covoiturage-libre/covoiturage-libre/wiki/Dev-setup-on-Ubuntu as the acceptance of this PR would further simplify the installation process described there. Again, I'm a ruby/rails noob so I may have done it wrong, please teach me :)

### CODE REVIEW

**Comment by Poussinou** [comment]
I think I will have the city list in a few days (I really hope!) Maybe we can wait for it.
Anyway, thanks for contributing, maybe @nicolasleger can review this PR?
Thanks for the wiki too :)

**Comment by webplate** [comment]
@Poussinou I really hope too :) And of course we should wait for it !
And thanks for motivating a nice reviewer !
@nicolasleger : I would be glad to have your opinion but don't worry if you don't feel like it, there's no rush, I'm here to learn.

@both My account has just been flagged !? I have no idea why... Did it ever happen to you ? What's the remedy ?

File: config/environments/development.rb [hunk 1]
```diff
@@ -29,6 +29,11 @@
     config.cache_store = :null_store
   end
 
+  # Set mailer host and port
+  ENV['MAILER_HOST'] = 'localhost:3000'
```
Comment (nicolasleger) [inline config/environments/development.rb]: Better to not overwrite ENV vars but use something like :
```
host = ENV['MAILER_HOST'] || 'localhost:3000'
# And then use host
```

File: db/seeds/cities.seeds.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  ['Lyon', '69000', '69', 'xxx', 'fr', 45.759723, 4.842223, 1],
+  ['Paris', '75000', '75', 'xxx', 'fr', 48.856578, 2.351828, 1],
+  ['Angers', '49000', '49', 'xxx', 'fr', 47.47361, -0.55416, 1],
+  ['Lille', '59000', '59', 'xxx', 'fr', 50.637222, 3.063333, 1],
+  ['Toulouse', '31000', '31', 'xxx', 'fr', 43.604482, 1.443962, 1],
+  ['Marseille', '13001', '13', 'xxx', 'fr', 43.296346, 5.369889, 1]
+]
+
+city_list.each do |name, postal_code, department, region, country_code, lat, lon, distance|
+  City.create(name: name, postal_code: postal_code, \
```
Comment (nicolasleger) [inline db/seeds/cities.seeds.rb]: Might be better to make it re-entrant via :
`City.where(postal_code: postal_code).first_or_initialize.update(name: name, ...)`

**Comment by nicolasleger** [comment]
Good @webplate ! I just made some comments via a Review.

It could be a good idea to merge it with https://github.com/covoiturage-libre/covoiturage-libre/pull/160

**Comment by webplate** [comment]
Wow, I didn't hope for so much :D 
Thank so much @nicolasleger ... I really am new to this world.
I also think #160 is important and related as it allows replication (an essential free software freedom).
I will rework my PR and then maybe we could merge :)

**Comment by nicolasleger** [comment]
No problem @webplate we're all learning, Ruby on Rails in my main stack near all day long. 🤓 
I'm glad to share and I'm available if you need help.
Thanks a lot for your active contributions

**Comment by webplate** [comment]
@thb as you are the initiator and the main contributor, your advice is precious, do you see any wrongdoing in my related tutorial ?
https://github.com/covoiturage-libre/covoiturage-libre/wiki/Dev-setup-on-Ubuntu
@nicolasleger I hope my contributions are not only active but also pertinent :) thanks for all the teachings !!

File: db/seeds/cities.seeds.rb (hunk 2 above)
Comment (webplate) [inline db/seeds/cities.seeds.rb]: I'm not sure about using the postal code in `where(`... in fact many locations in the world don't have a single postal code but a range of them. And it is hard to have all these ranges in the free geonames DB.

**Comment by webplate** [comment]
@nicolasleger I've chosen to check for name and region instead.

**Comment by webplate** [comment]
I think this is ready. No need to merge with #160 beforehand in my opinion (though it is fun).

**Pull request merged**